nautobot/docs/user-guide/administration/upgrading/from-v1/tables/v2-code-nautobot-app-location.yaml

```diff
@@ -0,0 +1,207 @@
+---
+- Nautobot Location: "nautobot.core.signals.nautobot_database_ready"
+  Public Interface: "nautobot.apps.nautobot_database_ready"
```

Comment:
This is going to be a pain to keep updated - every time we add something new to nautobot.apps, or relocate it behind the scenes, we will need to update this? I'm just not sure the value is here.

Reply:
@itdependsnetworks Is the intent that this data table is used to build a pylint rule? If so, since we have moved on in time, perhaps this one just moves solely into pylint?

Reply:
> I'm just not sure the value is here.

There is well over 100 known nautobot apps that should be able to benefit. 

> perhaps this one just moves solely into pylint?

Seems like a reasonable solution, I do think there should be a link to the list, as my process was to just pull up this file and do a ctrl-f and that was rather helpful.

Reply:
I missed this, thank you sir!!! (and no one would ever say that about you!!)